testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8 import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        
+        self.initial_car_order = [1, 2, 0, 3]
+        self.final_car_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(list(self.initial_car_order), list(self.final_car_order))

```

Comment:
If, however, we don't intend to check whether `self.initial_car_order` and `self.final_car_order` are modified in the production code, then there is probably no need to list stuffs XD